source/Government.cpp

```diff
@@ -466,9 +469,10 @@ double Government::AttitudeToward(const Government *other) const
 		return 1.;
 
 	if(attitudeToward.size() <= other->id)
-		return 0.;
+		return defaultAttitude;
```

Comment:
There's another one at the start of the function.

Reply:
isn't that a guard to make sure `other` is not a nullptr?

Reply:
Yeah. It should probably return the default attitude, because a null government is also a different government? I guess it doesn't really matter, because I just checked and the function is never called with a null government.